wg-charter.html

```diff
@@ -0,0 +1,364 @@
+        <h2>Scope</h2>
+        <p>
+            This Working Group will recommend a Web programming interface for
+            graphics and computation that:
+        </p>
+
+        <ol>
+            <li>enables rendering of modern graphics to both onscreen and offscreen drawing surfaces</li>
+            <li>enables computation tasks to be performed, and the results of such tasks to be retrieved</li>
+            <li>defines a Shading Language that describes the graphics and computation tasks in a format
+            that can be translated or compiled into platform-specific instructions</li>
+        </ol>
+
+        <p>
+            The API will not be restricted to any particular platform technology. Instead, it will
+            be generic enough to be implemented on top of modern GPU libraries, such as
+            Microsoft's Direct3D 12, Apple's Metal, and Khronos's Vulkan.
+        </p>
+
+        <p>
+            Note that the majority of the input to this Working Group will come
+            directly from the <a href="https://www.w3.org/community/gpu/">GPU for the Web Community Group</a>.
+            No major development will happen in the Working Group itself. Instead, the Community
+            Group will be driving the technical work.
+        </p>
+
+        <div>
+          <h3>Success Criteria</h3>
+          <p>In order to advance to  <a href="https://www.w3.org/2018/Process-20180201/#RecsCR" title="Proposed Recommendation">Proposed Recommendation</a>, each specification is expected to have at least two independent implementations of each of feature defined in the specification.</p>
+          <p>Each specification should contain a section detailing any <a href='https://www.w3.org/TR/security-privacy-questionnaire/'>known security</a>, fingerprinting, and privacy implications, and suggested <a href='https://w3c.github.io/perf-security-privacy/'>mitigation strategies</a> for implementers, Web authors, and end users. The group should not publish a specification if acceptable mitigation strategies cannot be found.</p>
+          <p>Normative specification changes are generally expected to have a corresponding set of tests, either in the form of new tests or modifications to existing tests, or must include the rationale for why test updates are not required for the proposed update.</p>
+        </div>
+      </section>
+
+
+
+      <section id="deliverables">
+        <h2>
+          Deliverables
+        </h2>
+
+        <div id="normative">
+          <h3>
+            Normative Specifications
+          </h3>
+          <dl>
+          <dt>WebGPU API</dt>
+
+          <dd>
+              An API as described by the <a
+              href="https://www.w3.org/community/gpu/">GPU for the Web
+              Community Group</a> charter.
+              <p class="milestone"><b>Recommendation expected completion:</b> Q2 2020</p>
+          </dd>
+
+          <dt>WebGPU Shading Language (WHLSL)</dt>
```

Comment:
This should be an optional deliverable, see comment above. Also I wouldn't callout the name WHLSL in this charter.

Reply:
✅